### PR TITLE
fix travis typo

### PR DESCRIPTION
**Pull request by mjenczmyk**
I've just changed `tracis` -> `travis` and made it a link. There are no more `tracis` misspellings.

### CODE REVIEW

**Review by jotes — APPROVED** [review]

**Comment by mjenczmyk** [comment]
Can it be merged/rejected or are we waiting for something?

**Comment by jotes** [comment]
@mjenczmyk I approved your PR after you requested review. We both have (or should have) equal rights to this repo, I thought you'll just merge this in your free time.

**Comment by jotes** [comment]
Funny, I just noticed that I don't see any merge button... Maybe I was wrong about out rights to this repo. @fizyk?

**Comment by mjenczmyk** [comment]
I was addressing @fizyk rather than you @jotes, maybe I should call him instead of asking everybody. Ayway, thanks.

**Comment by fizyk** [comment]
@jotes can you see the mnerge button now?

**Pull request merged**
